index.html

```diff
@@ -204,55 +204,1951 @@ <h3>
+        </ol>
+      </section>
+      <section>
+        <h3>
+          <dfn>requestDevice()</dfn> method
+        </h3>
+        <p>
+          The {{HID/requestDevice()}} method steps are:
+        </p>
+        <ol>
+          <li>Let |promise:Promise| be [=a new promise=].
+          </li>
+          <li>If [=this=]'s [=relevant global object=]'s [=associated
+          Document=] is not [=allowed to use=] the [=policy-controlled
+          feature=] named `"hid"`, [=reject=] |promise| with a
+          "{{SecurityError}}" {{DOMException}} and return |promise|.
+          </li>
+          <li>If the [=relevant global object=] of [=this=] does not have
+          [=transient activation=], [=reject=] |promise| with a
+          "{{SecurityError}}" {{DOMException}} and return |promise|.
+          </li>
+          <li>If |options|["{{HIDDeviceRequestOptions/filters}}"] is present,
+          then run the following steps for each |filter:HIDDeviceFilter| in
+          |options|["{{HIDDeviceRequestOptions/filters}}"]:
+            <ol>
+              <li>If |filter|["{{HIDDeviceFilter/productId}}"] is present and
+              |filter|["{{HIDDeviceFilter/productId}}"] is not present,
+              [=reject=] |promise| with a {{TypeError}} and return |promise|.
+              </li>
+              <li>If |filter|["{{HIDDeviceFilter/usage}}"] is present and
+              |filter|["{{HIDDeviceFilter/usagePage}}"] is not present,
+              [=reject=] |promise| with a {{TypeError}} and return |promise|.
+              </li>
+            </ol>
+          </li>
+          <li>Run the following steps [=in parallel=]:
+            <ol>
+              <li>Initialize |availableDevices| to be an empty [=list=].
+              </li>
+              <li>Repeat for each |device:HIDDevice| in
+              |navigator:Navigator|.{{Navigator/[[hidDevices]]}}:
+                <ol>
+                  <li>If |device| [=matches any filter=] in
+                  |options|["{{HIDDeviceRequestOptions/filters}}"], then append
+                  |device| to |availableDevices|.
+                  </li>
+                </ol>
+              </li>
+              <li>Prompt the user to grant the site access to a HID device by
+              presenting them with the list of devices in |availableDevices|.
+              </li>
+              <li>If the user does not choose a device, [=queue a global task=]
+              on the [=relevant global object=] of [=this=] using the [=HID
+              device task source=] to [=reject=] |promise| with an
+              {{"AbortError"}} {{DOMException}} and abort these steps.
+              </li>
+              <li>Let |device:HIDDevice| be the {{HIDDevice}} instance chosen
+              by the user.
+              </li>
+              <li>[=Queue a global task=] on the [=relevant global object=] of
+              [=this=] using the [=HID device task source=] to [=resolve=]
+              |promise| with |device|.
```

Comment:
`getDevices()` is supposed to return a sequence of `HIDDevice`.

Reply:
Oops, yes. Fixed.